tests/unit/test_column_noise.py

```diff
@@ -272,7 +261,38 @@ def string_series():
     )
 
 
-def test_leave_blank(dummy_dataset, fuzzy_checker: FuzzyChecker):
+@pytest.fixture()
+def dataset_data(dummy_dataset):
+    # We can't have this be a session scope because then we will be operating on the same object
```

Comment:
Ouch. Doesn't this kill the test runtime?

Reply:
No because I can copy the dummy_dataset. If I dont make dummy_dataset a session scope it does.